scripts/installer/machine/x86_64-accton_csp7551-r0/platform.conf

```diff
@@ -0,0 +1,37 @@
+GRUB_CMDLINE_LINUX="console=tty0 console=ttyS0,115200n8"
+GRUB_SERIAL_COMMAND="serial --port=0x3f8 --speed=115200 --word=8 --parity=no --stop=1"
+EXTRA_CMDLINE_LINUX=""
+BISDN_LINUX_VOLUME_SIZE_MB=163840
+
+platform_setup() {
+    (cat <<EOF
+[Match]
+Path=pci-0000:18:00.0
+
+[Link]
+Description=Tofino ICE port
```

Comment:
Can we give these a bit better description? What exactly is this? What does ICE mean? What's the difference to the Tofino CPU port? And also maybe add a `#0` and `#1` or so, so they have unique descriptions.

Reply:
Not sure what you expect here. I amended the description.

Reply:
I'm looking here for a clearer description on how these differ. You call all three of them CPU ports, but so are they functionally equivalent?